src/main/groovy/com/yodle/vantage/GenerateVantageJsonTask.groovy

```diff
@@ -34,6 +34,15 @@ class GenerateVantageJsonTask extends DefaultTask implements ReportProvider
 {
   public static final String DEFAULT_PROFILE = "default"
   public static final String LATEST_VERSION = "latest"
+  def doNotResolveConfigurations =
```

Comment:
https://discuss.gradle.org/t/3-4-rc-1-3-3-resolving-configurations-may-be-disallowed-and-throw-illegalstateexception/21470